### DISCUSSION TITLE
While updating the ODK form it is giving a version related error

### DISCUSSION THREAD

**Discussion by Nik720** [Samagra-Development/SamagraX-community#21] [Troubleshooting and Clarifications]
@SaketSinha @ChakshuGautam I am not able to update the odk form in the odk server. It is giving me a version-related error. I am uploading the form with the new version still it is failing

**FormID**:  sunbird_usecases 
**Existing version**: 2021033001
**New Version**: 2021033003

<img width="500" alt="Screenshot 2021-04-01 at 10 33 28 AM" src="https://user-images.githubusercontent.com/33539271/113246498-bf980600-92d6-11eb-9c77-3ecb1ea77ba2.png">

cc @vinukumar-vs @vrayulu

**Comment by aayushkedia95** [comment]
Hey @Nik720, can you share the excel of the ODK form here?

**Reply by Nik720** [reply]
@aayushkedia95  Here is the excel file of ODK form 
https://drive.google.com/file/d/1dKjmNx596QKzMAOgde67DDhMnzmr1eBJ/view?usp=sharing

**Reply by aayushkedia95** [reply]
Looking into this.

**Reply by aayushkedia95** [reply]
@Nik720, can you share all versions of the forms uploaded by you?

**Reply by Nik720** [reply]
@aayushkedia95 I have only the previous version(**2021033001**) backup
 
[sunbird_tara.xlsx](https://github.com/Samagra-Development/SamagraX-community/files/6257037/sunbird_tara.xlsx)

**Reply by aayushkedia95** [reply]
Hey @Nik720, Since the questions set in the previous form is different compared to the one you're uploading now, it is throwing this error. 

Versioning, apparently, only works if you update an existing question or add/update/delete related choices. 

Please upload a new form. That should work.
